pruntime_rpc.proto

```diff
@@ -136,6 +136,10 @@ message SystemInfo {
   uint64 number_of_clusters = 5;
   // The number of contracts deployed
   uint64 number_of_contracts = 6;
+  // The consensus version used to coordinate the pruntime's behavior
+  uint32 consensus_version = 7;
```

Comment:
Does it mean the current activated version on the blockchain, while the "max" means the version supported by this binary?

Reply:
Yep.

Reply:
Would love to add the explanation to the comment